assets/.sprockets-manifest-af5b3e33562477ba4f207e3ea1e798c0.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Should we remove this?

Reply:
This is autogenerated by jekyll. There's some sorcery in the Jekyll/bootstrap asset rendering that I intend to clean up once https://lists.apache.org/thread.html/73e8a7d2ee667e83371e8bb99c13338144c8af5444f55918994713be@%3Cdev.arrow.apache.org%3E is resolved.